inc/template-tags.php

```diff
@@ -330,24 +330,16 @@ function responsive_term_links( $post = null, $before = '', $sep = '', $after =
  * @uses  BU Navigation plugin
  */
 function responsive_primary_nav() {
-	if ( ! method_exists( 'BuAccessControlPlugin', 'is_site_403' ) ||
-		false == BuAccessControlPlugin::is_site_403() ) {
-
-		if ( function_exists( 'bu_navigation_display_primary' ) ) {
-			bu_navigation_display_primary( array(
-				'container_id'    => 'primary-nav-menu',
-				'container_class' => 'primary-nav-menu',
-			) );
-		} else {
-			wp_nav_menu( array(
-				'theme_location' => 'responsive-primary',
-				'menu_id'        => 'primaryNav-menu',
-				'menu_class'     => 'primaryNav-menu',
-				'container_tag'  => 'ul',
-				'depth'          => 2,
-			) );
-		}
-	}
+	$nav = apply_filters( 'responsive_primary_nav', wp_nav_menu( array(
```

Comment:
Do we want to include `wp_nav_menu` in that filter? or just the array of `$args`?

I was thinking something like 

```
$args = apply_filters( 'responsive_primary_nav_args', array(
..
);
```

and then 

```
echo wp_nav_menu( $args );
```

And if child themers don't want this function to fire they could overwrite it and return false, if we wrapped it with `if ( ! function_exists( 'responsive_primary_nav' ) )`

Thoughts?